### DISCUSSION TITLE
Docs Not Working: https://doc.primekey.com/

### DISCUSSION THREAD

**Discussion by ToddBerliner** [Keyfactor/ejbca-ce#404] [Q&A] [answered]
I can't access any docs related links. Am I missing something?

**Comment by ToddBerliner** [comment, marked as answer]
For example: https://doc.primekey.com/ejbca/tutorials-and-guides/quick-start-guide-start-ejbca-container-with-client-certificate-authenticated-access

**Reply by ToddBerliner** [reply]
Docs are down due to IT issue according to other discussion

**Reply by primetomas** [reply]
https://doc.primekey.com/ is up again
